### ISSUE TITLE
Api broken?

### ISSUE THREAD

**Issue by nicoduck** [peeringdb/peeringdb-py#33]
Hi,

I have some issues with the refactoring and the documentation, for example:
_pdb.all(resource.Network, asn=2906)_ is listed as an example in the documentation but throws an error nowadays:
```
>>> pdb.all(resource.Network, asn=2906)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: all() got an unexpected keyword argument 'asn'
```

I assume that my setup works completely since  _pdb.get(resource.Network, 16954)_ returns the expected network.
`>>> pdb.get(resource.Network, 16954)
<Network: Pfalzkom>
`
It also seems that the [presentation](https://docs.peeringdb.com/presentation/PeeringDB_Version_2-Coding_Introduction.pdf) about PeeringDB 2 refers to the old library implementation as _pdb.all('ix', name='chix', country='us')_ does not seem to work anymore too.

Could you please clarify how the syntax has changed, how to get the expected results and update the documentation?
I'm running python 3.6.8 and _peeringdb --version_ returns:
peeringdb 1.0.0

Thanks
Nico

**Comment by svaio** [comment]
try to use pdb.fetch_all(resource.Network, 999, asn=2906)

**Comment by arnoldnipper** [comment]
did that fix your problem, @nicoduck?

**Comment by nicoduck** [comment]
Yes, but that also means that the documentation seems to be outdated.
fetch_all is not listed, as is "import resource from peeringdb", which is necessary to execute it.

**Comment by arnoldnipper** [comment]
Would you mind to list the URL, @nicoduck you are referring to?

**Comment by nicoduck** [comment]
I'm referring to the docs linked in the Readme (aka the startpage) of the project: http://peeringdb.github.io/peeringdb-py/api/

**Comment by arnoldnipper** [comment]
@vegu | @grizz could you please look into the documentation. TIA

**Comment by toomuchio** [comment]
fetch_all avoids the local database and requests data directly from the API, is that intentional?

**Comment by grizz** [comment]
> fetch_all avoids the local database and requests data directly from the API, is that intentional?

Yes, fetch is fetching from server.

**Comment by vegu** [comment]
this issue is referencing code that has since changed significantly as no longer relevant, closing
